app/tests/form-freelance/integration-test/formDisplay.test.tsx

```diff
@@ -0,0 +1,10 @@
+import React from 'react'
+import { render, screen } from '@testing-library/react'
+import FormFreelance from '../../../pages/formulaire-freelance'
```

Comment:
It's more maintainable to use project root routes here : 

```suggestion
import FormFreelance from '@/pages/formulaire-freelance'
```
if @ does what i think it does (I saw it but I havent tried yet)

Reply:
i need to setup this "@"  idk if it is native to nextjs